_generated/NuGetRestoreV1_Node20/nugetinstaller.ts

```diff
@@ -18,294 +18,8 @@ import * as telemetry from "azure-pipelines-tasks-utility-common/telemetry";
 const NUGET_ORG_V2_URL: string = "https://www.nuget.org/api/v2/";
 const NUGET_ORG_V3_URL: string = "https://api.nuget.org/v3/index.json";
```

Comment:
Unused `const`s and imports?

Reply:
Removed these since they're no longer needed!